### ISSUE TITLE
Thunderbird 78.4.3 compatibility

### ISSUE THREAD

**Issue by t-oster** [bmlk/filemail#2]
Hi,
I really like your plugin. However my disto just updated thunderbird and the plugin won't run anymore. Any plans to update?

**Comment by ghost** [comment]
yup, would be great to have an update!! I also miss this great addon!

**Comment by ghost** [comment]
ping!

**Comment by bmlk** [comment]
Yes, I understand this, but between your  wishes and  a new version of the plugin  stands a complete mess (or better a absence of) in the documentation, that makes every step here for me very difficult.  I would be grateful for help. I will probably push what I have and mark the difficulties that I see.

**Comment by ghost** [comment]
happy to make a small donation if that helps

**Comment by t-oster** [comment]
me too

**Comment by ghost** [comment]
any update on this?
happy to help financially if needed

**Comment by t-oster** [comment]
@bmlk I would also be willing to make a small donation (10$?) if that helps. I also use this plugin https://github.com/kewisch/quickmove-extension, which is a great addition to yours. Since this was updated recently, maybe you can look at the code, because at least for the moving part you should be using the same API.

**Comment by bmlk** [comment]
I just pushed a new web extension version which I am going to submit for review
